newIDE/app/src/Profile/SubscriptionChecker.js

```diff
@@ -128,9 +127,8 @@ export class SubscriptionCheckDialog extends React.Component<
             <Star style={styles.icon} />
             <Text style={styles.iconText}>
               <Trans>
-                Having a subscription allows you to use the one-click export for
-                Android, Windows, macOS and Linux, launch live previews over
-                wifi, disable the GDevelop splashscreen during loading and more!
+                Having a subscription allows you to use the one-click export and
+                disable the GDevelop splashscreen during loading!
```

Comment:
yes and we can also add that take a plan remove the message before some features.